release-please-config.json

```
@@ -0,0 +1,62 @@
+{
+    "bootstrap-sha": "e44f9f26670de2b11336009a988e86bb4cd32676",
+    "packages": {
+        ".": {
```

I have a feeling this might need to be changed to something like the following, but I'm not entirely sure... 

```suggestion
        "./charts/flagsmith": {
```

---

I went ahead and did this (or similar at least). I think we'll just need to test it out and see how it behaves. The documentation doesn't confirm if the `changelog-path` should be relative to the root, or to the package, but chatgpt seems to think it's the former, so I've stuck with just `"CHANGELOG.md"` for that.